internal/server/server.go

```
@@ -34,16 +38,20 @@ import (
 // DefaultAddr is where the daemon listens.
 const DefaultAddr = "localhost:7777"
 
+// claudeProjectsRootEnv overrides the Claude transcripts root path.
+const claudeProjectsRootEnv = "CLAUDE_PROJECTS_ROOT"
```

golangci-lint (2.12.1)

[error] const claudeProjectsRootEnv is unused

(unused)